Cargo.toml

```diff
@@ -15,8 +15,8 @@ path = "src/passthrough.rs"
 tokio = { version = "1.17.0", features = ["full"] }
 futures-util = { version = "0.3.21" }
 
-zeroconf = { git = "https://github.com/zeyugao/zeroconf-rs", optional = true }
-mdns = "3.0.0"
+zeroconf = { version = "0.14.2", optional = true }
+mdns = { version = "3.0.0", optional = true }
```

Comment:
Does the crate still work with the mdns dependency?

Reply:
Removed mdns crate.
commit 57b380b98e4b8552bf96a655921ad88fbd9a2268